target-app/Home/Views/HomeContainerViewController.swift

```diff
@@ -0,0 +1,36 @@
+//
+//  HomeContainerViewController.swift
+//  target-app
+//
+//  Created by Andres Leonel Torrico Cossio on 12/09/2022.
+//
+
+import UIKit
+
+final class HomeContainerViewController: BottomSheetContainerViewController <UIViewController, UIViewController> {
+    
+    override func viewDidAppear(_ animated: Bool) {
+        setupNavigationBar()
+    }
+    
+    func setupNavigationBar() {
+        navigationController?.setNavigationBarHidden(false, animated: true)
+        
+        navigationItem.titleView = UILabel(style: .secondary(text: "home_title".localized))
+        
+        let barLeftButtonItem = UIBarButtonItem(image: UIImage(named: "ic_home_profile"),
+                                                style: .plain,
+                                                target: self,
+                                                action: nil)
```

Comment:
Pls check the indentation style in our Swift Style guide

Reply:
done! thanks!